docs/zh/reference/plugin/docsearch.md

```diff
@@ -89,18 +89,18 @@ npm i -D @vuepress/plugin-docsearch@next
   从 DocSearch 团队收到的 `indexName` ，或者由你自己生成。
 
 - 参考：
-  - [DocSearch > Options > indexName](https://autocomplete-experimental.netlify.app/docs/docsearch-js#indexname)
+  - [DocSearch > Options > indexName](https://docsearch.algolia.com/docs/api#indexname)
 
 ### appId
 
 - 类型： `string`
 
 - 详情：
 
-  只有在你 [运行你自己的 DocSearch 爬虫](https://docsearch.algolia.com/docs/run-your-own/) 时，才需要配置该选项。用于设置你的 Application ID 。
+  只有在你 [运行你自己的 DocSearch 爬虫](https://docsearch.algolia.com/docs/legacy/run-your-own/) 时，才需要配置该选项。用于设置你的 Application ID 。
```

Comment:
```suggestion
  在你 [运行你自己的 DocSearch 爬虫](https://docsearch.algolia.com/docs/legacy/run-your-own/) 或已经 [迁移到新的 DocSearch 结构](https://docsearch.algolia.com/docs/migrating-from-legacy) 时需要配置此选项。用于设置你的 Application ID。
  
```

Reply:
Hey, thanks! Can you also submit a change on the v1? https://github.com/vuejs/vuepress/pull/2957